.ado/pipelines/templates/jobs-configuration.yaml

```diff
@@ -106,16 +106,23 @@ jobs:
           echo "*** Apply cert-manager CRDs on $aksClusterName"
           kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$(certManagerVersion)/cert-manager.crds.yaml
 
-          # Preparations to deploy cert-manager into it's own cert-manager namespace
-          echo "*** Create a separate namespace for cert-manager on $aksClusterName"
-          kubectl apply -f src/config/cert-manager/cert-manager.yaml
+          $certManagerNamespace = "cert-manager"
+
+          echo "*** Create namespace '$certManagerNamespace' on AKS cluster $aksClusterName via kubectl"
+          kubectl create namespace $certManagerNamespace `
+                    --dry-run=client --output yaml | kubectl apply -f -
+
+          echo "*** Setting up cert-manager ClusterIssuer with Let's Encrypt"
+          helm upgrade --install cert-manager-alwayson src/config/charts/cert-manager `
```

Comment:
would it make sense to call this chart `cert-manager-config` instead of `cert-manager-alwayson`?